### DISCUSSION TITLE
Google auth is not working

### DISCUSSION THREAD

**Discussion by mobcoder-kishan-walecha** [ionic-team/capacitor#5231] [General]
Hello Team,

I am implementing google authentication through capacitor in android app, I've used @codetrix-studio/capacitor-google-auth library as capacitor/core has removed 'plugin' from its current version and I am getting below error -

**E/Capacitor/Plugin: Something went wrong
    com.google.android.gms.common.api.ApiException: 10:** 

I have tried to find some online solution and I got that I have miss-configured the SHA-1 but I have followed all approaches to configure the SHA-1 (from keytool command and from android studio) but the same error persist.
It will be really helpful if someone can provide me steps to configure it properly.

**Comment by bphd** [comment]
E/Finsky  (): [173] uks.a(849): Encountered exception whilst fetching from GMS, deferring to Finsky: com.google.android.gms.common.api.ApiException: 17: API: IntegrityApiDisplayListener.API is not available on this device. Connection failed with: ConnectionResult{statusCode=API_DISABLED, resolution=null, message=null}
